.htaccess

```diff
@@ -0,0 +1,32 @@
+<IfModule mod_rewrite.c>
+        SetEnv HTTP_MOD_REWRITE On
+        RewriteEngine On
+
+        # "Symlink" the root to the latest release (with "moved temporarily")
+        RewriteRule ^/?$ /srcdocs/3.6/ [R=302,L]
+
+        # Remove trailing slashes, ensures the redirect to srcdocs works
+        RewriteCond %{REQUEST_FILENAME} !-d
+        RewriteRule ^(.*)/$ /$1 [L,R]
+
+        # Lookup script (used to be a SS app, now just a simple script)
+        RewriteCond %{REQUEST_URI} ^/search/lookup$
+        RewriteRule ^search/lookup /search/lookup.php [L,R=301,QSA]
+
+        # "Symlink" a "current" folder to the latest release
+        RewriteRule ^.*/current.*?(.*)? /srcdocs/3.6/$1 [R=301,L]
+
+        # Rewrite trunk to master
+        RewriteRule ^.*/trunk/.*(.*)? /srcdocs/master/$1 [L,R=301]
```

Comment:
Ok, well I'm fine with us using `/en/` as the prefix since that'd match the other docs sites etc. We'll need rewrite rules for the existing routes though, e.g. `/3.6/` -> `/en/3.6/` etc so we don't get dead cross linking